plugins/outputs/websocket/websocket.go

```diff
@@ -92,7 +92,14 @@ func (w *WebSocket) Connect() error {
 
 	headers := http.Header{}
 	for k, v := range w.Headers {
-		headers.Set(k, v)
+		secret, err := v.Get()
+		if err != nil {
+			return err
+		}
+
+		headerVal := secret.String()
+		headers.Set(k, headerVal)
```

Comment:
Purely cosmetic but I think this looks better:
```suggestion
		headers.Set(k, secret.String())
```